### PR TITLE
mock: create local config server, HTTP server, and client

### PR DESCRIPTION
**Pull request by cmnord**

### CODE REVIEW

File: mock/config.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	"net"
+	"net/http"
+	"path/filepath"
+
+	"vuvuzela.io/alpenhorn/config"
+)
+
+// ClientAndServer creates a local config server, a config HTTP server, and a
+// config client.
+func ClientAndServer(dir string) (*config.Server, *config.Client) {
```
Comment (davidlazar) [inline mock/config.go]: Rename to `LaunchConfigServer` to match other mock package.

Comment (cmnord) [reply mock/config.go]: amended

Comment (cmnord) [reply mock/config.go]: Addressed in https://github.com/vuvuzela/alpenhorn/pull/9 as well

**Comment by davidlazar** [comment]
Does the next PR supersede this one?

**Comment by cmnord** [comment]
Yes, this is a subset of the changes in #2

**Pull request closed without merging**